app/models/in_progress_form.rb

```diff
@@ -19,8 +20,25 @@ def cast(value)
   before_save :serialize_form_data
   before_save :set_expires_at
 
+  scope :for_user, lambda { |user|
+                     where(user_uuid: user.uuid)
+                       .or(where(user_uuid: ACCT_ID_PREFIX + user.account_id))
+                   }
+
   def self.form_for_user(form_id, user)
-    InProgressForm.find_by(form_id: form_id, user_uuid: user.uuid)
+    InProgressForm.for_user(user).find_by(form_id: form_id)
+  end
+
+  # Resolve a form given an "indifferent" user_id, i.e.
+  # one that may be either an idme_uuid or a prefixed account_id
+  def self.form_for_user_id(form_id, user_id)
+    acct = Account.find_by(idme_uuid: user_id)
+    acct = Account.find_by(uuid: user_id.delete_prefix(ACCT_ID_PREFIX)) if user_id.start_with?(ACCT_ID_PREFIX)
```

Comment:
```suggestion
    args = user_id.start_with?(ACCT_ID_PREFIX) ? { uuid: user_id.delete_prefix(ACCT_ID_PREFIX) } : { idme_uuid: user_id }
    acct = Account.find_by(args)
```

Reply:
Yup that's better, Rubocop had a multi-line ternary complaint that i was trying to get rid of and my mind went in a different direction